js/templates/userPage/userPageHome.html

```diff
@@ -0,0 +1,81 @@
+<div class="flexRow gutterH">
+  <div class="col4 gutterVMd">
+    <div class="js-userShort">
+      <!-- the userShort template is injected here-->
+    </div>
+    <% if(ob.isModerator) { %>
+    <div class="contentBox padMd2 clrBr clrP clrSh2 js-moderatorInfo">
+      <div class="flexVBot gutterH rowLg snipKids">
+        <div class="tx4 flexExpand"><strong><%= ob.polyT('userPage.moderator') %></strong></div>
+        <div class="clrT2 tx5">
+          <a class="link js-termsLink">
+            <%= ob.polyT('userPage.termsOfService') %>
+          </a>
+        </div>
+      </div>
+      <div class="tx2 clrT2 txCtr rowMd"><%= ob.polyT('userPage.modFee', {modFee: 1.3}) %></div>
+      <div class="flexHCent">
+        <a class="btn clrP clrBr clrSh2 js-addModerator">
+          <span class="js-addModeratorLbl <% if(ob.currentModerator) print('hide') %>">
+            <%= ob.polyT('userPage.addModerator') %>
+          </span>
+          <span class="js-removeModeratorLbl <% if(!ob.currentModerator) print('hide') %>">
+            <%= ob.polyT('userPage.removeModerator') %>
+          </span>
+        </a>
+      </div>
+    </div>
+    <% } %>
+    <div class="informationList listBox clrBr clrP clrSh2">
+      <div class="informationHeader">
+        <div><%= ob.polyT('userPage.information') %></div>
+      </div>
+      <div class="listItem">
+        <div><%= ob.polyT('userPage.website') %></div>
+        <div class="clrT2"><a href="<%= ob.website %>"><%= ob.website %></a></div>
+      </div>
+      <div class="listItem">
+        <div><%= ob.polyT('userPage.email') %></div>
+        <div class="clrT2"><%= ob.email %></div>
+      </div>
+      <div class="listItem">
```

Comment:
I imagine the social accounts declared like this is just placeholder code... because I think we will want to switch it to build off of the nested social collection within Profile model.

In fact, if we want we could even make a separate little view for these guys and then just re-render on the `update` event of the collection. That way if someone had modified their settings while this page was in the background, it would be nice and updated for them without having to refresh the page.